gulpfile.js/import/importRoadmap.js

```diff
@@ -143,65 +138,56 @@ async function getMetaForWorkigGroup(workingGroup) {
 }
 
 async function getIssuesForWorkingGroup(meta) {
-  const issues = (
+  const issues = [];
+  const issuesImport = (
     await client._github
       .repo(`${DEFAULT_ORGANISATION}/${meta.name}`)
       .issuesAsync()
   )[0];
 
-  return issues
-    .filter((issue) => {
-      const category = issue.labels[0] ? issue.labels[0].name : '';
-      return ALLOWED_ISSUE_TYPES.includes(category);
-    })
-    .map((issue) => {
-      const createdAt = new Date(issue.created_at).toDateString();
-      const title = emojiStrip(issue.title);
-
-      // Escape amp-components in markdown to prevent them from being rendered as such
-      // plus remove Emojis and split into separate text blocks to allow smoother line breaks in frontend
-      let body = issue.body.replace(AMP_COMPONENT_REGEX, ' `$1`');
-      body = emojiStrip(body).trim().match(TEXT_BLOCK_REGEX);
-
-      // Parse status update date from from issue title and set quarter
-      let statusUpdate = title.match(STATUS_UPDATE_REGEX);
-      let quarter;
-      if (statusUpdate) {
-        quarter = `Q${Math.ceil(parseInt(statusUpdate[2]) / 3)} ${
-          statusUpdate[1]
-        }`;
-        statusUpdate = new Date(statusUpdate[0]).toDateString();
-      } else {
-        log.error(
-          `.. ${meta.slug} - Could not parse valid date from issue title: ${title}`
-        );
-        return;
-      }
+  for (const issue of issuesImport) {
+    const category = issue.labels[0] ? issue.labels[0].name : '';
+    if (!ALLOWED_ISSUE_TYPES.includes(category)) {
+      continue;
+    }
 
-      return {
-        'wg_slug': meta.slug,
-        'wg_title': meta.title,
-        'wg_color': meta.color,
-        'created_at': createdAt,
-        'status_update': statusUpdate,
-        'quarter': quarter,
-        'number': issue.number,
-        'html_url': issue.html_url,
-        'body': body,
-      };
-    })
-    .filter((issue) => issue);
-}
+    const createdAt = new Date(issue.created_at).toDateString();
+    const title = emojiStrip(issue.title);
+
+    // Escape amp-components in markdown to prevent them from being rendered as such
+    // plus remove Emojis and split into separate text blocks to allow smoother line breaks in frontend
+    let body = issue.body.replace(AMP_COMPONENT_REGEX, ' `$1`');
+    body = emojiStrip(body).trim().match(TEXT_BLOCK_REGEX);
+
+    // Parse status update date from from issue title and set quarter
+    let statusUpdate = title.match(STATUS_UPDATE_REGEX);
+    let quarter;
+    if (statusUpdate) {
```

Comment:
Move this block up to line 152. So that all your potential error cases are handled first - because if this fails you do not need to strip emojis from the title, replace the components in the body, ...